cpp/src/arrow/csv/parser.cc

```diff
@@ -144,7 +144,11 @@ class ValueDescWriter {
 
  protected:
   ValueDescWriter(MemoryPool* pool, int64_t values_capacity)
-      : values_size_(0), values_capacity_(values_capacity), status_(Status::OK()) {
+      : values_size_(0),
+        values_capacity_(values_capacity),
+        quoted_(false),
+        saved_values_size_(0),
+        status_(Status::OK()) {
```

Comment:
`quoted_(false)` and `saved_values_size_(0)` are the only additions here (the rest is whitespace)